docs/tuf-spec.txt

```diff
@@ -928,24 +928,120 @@ Version 1.0 (Draft)
 
 5.1. The client application
 
-   1. The client application first instructs TUF to check for updates.
+    1. **Download the root metadata file**, up to some X number of bytes
+    (because the size is unknown). The value for X is set by the authors of the
+    application using TUF. For example, X may be tens of kilobytes.
 
-   2. TUF downloads and verifies timestamp.json.
+    1.1. **Check signatures.** The root metadata file must have been signed by
```

Comment:
As for where to mention pinning vs. TOFU, I think there should be a section before 5.1 maybe called "client design" that mentions this, and then step 1 could specifically say something like "If a client was initialized with a pinned set of root keys, these keys and only these keys MUST be used to validate the original root.json."